vitrina/datasets/models.py

```diff
@@ -129,6 +129,33 @@ def get_absolute_url(self):
     def get_tag_list(self):
         return str(self.tags).replace(" ", "").split(',') if self.tags else []
 
+    def get_members_url(self):
+        return reverse('dataset-members', kwargs={'org_kind': str(self.organization.kind) if self.organization else None,
+                                                  'org_slug': self.organization.slug if self.organization else None,
+                                                  'slug': self.slug})
+
+    def dataset_member_set(self):
+        members = DatasetMember.objects.filter(dataset=self)
+        return members
+
+
+class DatasetMember(models.Model):
+    CREATOR = 'dct:creator'
+    CONTRIBUTOR = 'dct:contributor'
+    PUBLISHER = 'dct:publisher'
+
+    ROLE_CHOICES = (
+        (CREATOR, _('Creator')),
+        (CONTRIBUTOR, _('Contributor')),
+        (PUBLISHER, _('Publisher')),
+    )
+    user = models.ForeignKey(User, models.DO_NOTHING, blank=True)
```

Comment:
```suggestion
    user = models.ForeignKey(User, models.CASCADE)
```

Both `user` and `dataset` should be required and unique constraint should be put on both fields.